fdbserver/DataDistribution.actor.cpp

```diff
@@ -1637,6 +1663,9 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 			if (EXPENSIVE_VALIDATION) {
 				ASSERT(isServerTeamCountCorrect(mt));
 			}
+			if (mt->existServerWithOnly1Team()) {
```

Comment:
We do not want to remove a team which will have to be immediately rebuilt, so change this check to server with only DESIRED

Reply:
Done